spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BindingServiceProperties.java

```diff
@@ -127,9 +127,10 @@ public void setEnvironment(Environment environment) {
 		if (environment instanceof ConfigurableEnvironment) {
 			// override the bindings store with the environment-initializing version if in
 			// a Spring context
+			Map<String, BindingProperties> delegate = new TreeMap<String, BindingProperties>(String.CASE_INSENSITIVE_ORDER);
```

Comment:
Isn't the `bindings` always be `null` at this phase of the lifecycle? Referring the original issue from Spring Cloud Bus [here](https://github.com/spring-cloud/spring-cloud-bus/blob/master/spring-cloud-bus/src/main/java/org/springframework/cloud/bus/BusAutoConfiguration.java#L69), the `bindings` would be set only after the beanPostProcessor EnvironmentAware interfaces are invoked. I am trying to understand what impact does this change make by updating the `this.bindings` at this level.